README.md

```diff
@@ -1,43 +1,58 @@
-# eslint-import-resolver-kibana
+# @elastic/eslint-import-resolver-kibana
 
 Resolver for Kibana imports, meant to be used with [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import).
 
 ## Usage
 
-In your `.eslintrc.(yml|json|js)` file, add the following `import/resolver` under `settings`:
+Specify this resolver at `settings["import/resolver']` in your eslint config file:
 
 ```yml
 # .eslintrc.yml
 settings:
-  # uses 'eslint-import-resolver-kibana':
-  import/resolver:
-    - "@elastic/eslint-import-resolver-kibana"
+  import/resolver: "@elastic/eslint-import-resolver-kibana"
 ```
 
-The `node` resolver isn't strictly required, but the resolving rules are unlikely to work as you expect if you don't include it in the list of resolvers. It's only included by default if you don't specify other resolvers.
+## Settings
+
+> NOTE: all relative paths are resolved as relative to the project root, which is determined by walking up from the first linted file and looking for a `package.json` file. If you project has multiple `package.json` files then make sure to specify the `rootPackageName` setting.
+
+Property | Default | Descritpion
+-------- | ------- | -----------
+rootPackageName | `null` | The `"name"` property of the root `package.json` file
+kibanaPath | `../kibana` | Relative path to the kibana root
+pluginDirs | `[]` | Array of relative paths pointing to directories which contain Kibana plugins
```

Comment:
Add a note that each plugin requires its own `package.json` file to be included.